hoot-rnd/src/main/cpp/hoot/rnd/schema/ImplicitTagRawRulesDeriver.cpp

```diff
@@ -412,7 +416,7 @@ void ImplicitTagRawRulesDeriver::_sortByTagOccurrence()
     "sed -e 's/^ *//;s/ /\\t/' > " + _sortedCountFile->fileName();
   if (std::system(cmd.toStdString().c_str()) != 0)
   {
-    throw HootException("Unable to sort file.\n----------\n" + cmd + "\n----------\n");
+    throw HootException("Unable to sort file.");
```

Comment:
Put this in to see what was causing the `develop` build to fail last week.  It isn't needed now.